tests/floats/tests/floats.ts

```diff
@@ -0,0 +1,63 @@
+import * as anchor from "@project-serum/anchor";
+import { Program } from "@project-serum/anchor";
+import { Floats } from "../target/types/floats";
+import assert from "assert";
+
+describe("floats", () => {
+  // Configure the client to use the local cluster.
+  anchor.setProvider(anchor.Provider.env());
+
+  const program = anchor.workspace.Floats as Program<Floats>;
+
+  it("Creates an account with float data", async () => {
+    const accountKeypair = anchor.web3.Keypair.generate();
+
+    await program.rpc.create(1.0, 2.0, {
```

Comment:
Please change all new tests to use the more modern `program.methods` API. Other namespaces will eventually be deprecated.